js/client-chat.js

```diff
@@ -958,6 +958,13 @@
 					});
 					break;
 
+				case 'remove':
+					if (app.user.get('isStaff')) break;
+					$('.chatmessage-' + toId(row[1])).each(function() {
+						$(this).remove();
+					});
```

Comment:
Anyway, using a raw `$` is pretty unacceptable for something as complex as PS; we'd generally use `this.$` in place of that.